src/components/HookSubscriber/HookSubscriber.tsx

```diff
@@ -71,6 +72,8 @@ const HookSubscriber: FC = () => {
     subscribeSavingsYield();
     subscribeManageTransactions();
     subscribeSavingsTransactions();
+    subscribeBridgeTokensStatus();
```

Comment:
all these `subscribeXXX` exist becoz we want to have unit test cover them and to have a control to subscribe them instead of automatically subscribe during declaration time.
given that we dont have unit tests for the hooks, we can remove them and do the way you said.